resources/travis/linux_script.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3.7
+import sys
+import time
+import datetime
+
+from pathlib import Path
+from sultan.api import Sultan
+
+
+def main():
+    time_script_start = time.perf_counter()
+    examples_dir = Path("examples/connext_dds").resolve()
+    build_dir = examples_dir.joinpath("build")
+    cmake_build_all_path = Path(
+        "resources/cmake/ConnextDdsBuildAllConfigurations.cmake"
+    ).resolve()
+
+    if not examples_dir.exists():
+
+        sys.exit("Examples directory not found.")
+
+    if not cmake_build_all_path.exists():
+
+        sys.exit("Cmake script to build all configurations not found.")
+
+    print("Building the examples...", flush=True)
+    time_build_start = time.perf_counter()
+
+    build_dir.mkdir(exist_ok=True)
+
+    if not build_dir.exists():
+
+        sys.exit("Build dir not found.")
+
+    with Sultan.load(cwd=build_dir) as sultan:
+
+        result = sultan.cmake(
+            "-DBUILD_SHARED_LIBS=ON",
+            "-DCMAKE_BUILD_TYPE=Release",
+            examples_dir,
+        ).run(halt_on_nonzero=False)
+
+        if result.rc != 0:
+
+            sys.exit("There was some errors during build.")
+
+        result = sultan.intercept__build(
+            "cmake", "--build .", "--config Release"
+        ).run(halt_on_nonzero=False)
+
+        if result.rc != 0:
+
+            sys.exit("There was some errors during build.")
+
+        time_build_end = time.perf_counter()
+
+        print("Analyzing the build...", flush=True)
+        time_analysis_start = time.perf_counter()
+
+        result = sultan.analyze__build(
+            "--verbose",
+            "--status-bugs",
+            "--exclude",
+            Path.home().joinpath("rti_connext_dds-6.0.0/include"),
+            "--exclude",
```

Comment:
Is there a way to avoid hardcoding the version here? Maybe we can ask for the path to rti_connext_dds-x.y.z, which would solve the problem of having to pass a specific DDS version here.